xbmc/video/videosync/VideoSyncCocoa.cpp

```diff
@@ -76,10 +69,11 @@ void CVideoSyncCocoa::VblankHandler(int64_t nowtime, double fps)
   //update the vblank timestamp, update the clock and send a signal that we got a vblank
   UpdateClock(NrVBlanks, nowtime);
 }
+#endif
 
 bool CVideoSyncCocoa::Setup(PUPDATECLOCK func)
 {
-  CLog::Log(LOGDEBUG, "CVideoSyncCocoa: setting up Cocoa");
+  CLog::Log(LOGDEBUG, "CVideoSyncCocoa::%s - setting up Cocoa", __FUNCTION__);
   bool setupOk = false;
```

Comment:
This will look like `CVideoSyncCocoa::CVideoSyncCocoa::VblankHandler - ...`. in log. The __FUNCTION__ macro also includes class name.